smoketest/graphqlnotifiedalerts_test.go

```diff
@@ -0,0 +1,135 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/smoketest/harness"
+)
+
+// TestNotifiedAlerts tests that the alerts GraphQL query shows the proper amount of results when flipping between "includeNotified" and "favoritesOnly" query options.
+// Service 1: Notified alert, Service 2: Favorited alert
+
```

Comment:
Remove the blank line for the comment to be "attached" to the func

Reply:
Removed blank line.